### PR TITLE
[wip] glassfish 5.1.0

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by chenrui333** [comment]
```
09:03:03 ==> Installing 'bundler' gem
09:03:03 Successfully installed bundler-2.0.1
09:03:03 1 gem installed
09:03:03 Error: 1 problem in 1 formula detected
09:03:03 glassfish:
09:03:03   * C: 4: col 3: Use GitHub tarballs rather than zipballs (url is https://github.com/eclipse-ee4j/glassfish/archive/5.1.0.zip).
09:03:03 
09:03:03 ==> brew style glassfish
```

**Comment by chenrui333** [comment]
Thanks for the update change!

File: Formula/glassfish.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Glassfish < Formula
   desc "Open Source Jakarta EE Platform Implementation"
   homepage "https://eclipse-ee4j.github.io/glassfish/"
-  url "https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip&mirror_id=518"i
```
Comment (chenrui333) [inline Formula/glassfish.rb]: I dont think you can make that change, it missed the bin folder which is needed to boot up an application server for java apps.
image

File: Formula/glassfish.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Glassfish < Formula
-  desc "Java EE application server"
-  homepage "https://glassfish.java.net/"
-  url "https://download.oracle.com/glassfish/5.0/release/glassfish-5.0.zip"
-  sha256 "85450f0cd4875729d64be6bcbf7ed8f61009ea0f23d3be453c6f3d1dbab02420"
+  desc "Open Source Jakarta EE Platform Implementation"
+  homepage "https://eclipse-ee4j.github.io/glassfish/"
+  url "https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip&mirror_id=518"i
```
Comment (SMillerDev) [inline Formula/glassfish.rb]: What's the I doing there and should we specify a mirror?

**Comment by chenrui333** [comment]
@SMillerDev, I sort of understand your perspective now, the glassfish project has transferred the ownership, now it becomes painful to make the change, I am going to close it now. As I has been quite a while.

**Pull request closed without merging**